packages/platform/atoms/booker/BookerPlatformWrapper.tsx

```diff
@@ -126,7 +126,10 @@ export const BookerPlatformWrapper = (
     return "";
   }, [props.username]);
 
-  setSelectedDuration(props.duration ?? null);
+  useEffect(() => {
+    setSelectedDuration(props.duration ?? null);
+  }, [props.duration]);
```

Comment:
Why do we need useEffect here - if props.duration changes it should cause this to re-render also

Reply:
this is to ensure that the duration selected by the user of the booker is not overriden by the default duration given as props to the booker on every render